test-cases/gutenberg/color-settings.md

```diff
@@ -0,0 +1,183 @@
+
+# Color Settings - Test Cases
+
+#### **Precondition**
+
+Added block with color support such as `Buttons` or `Cover` 
```

Comment:
I think we should add the steps to open the color settings and remove "Press `Background Color` / `Color`" from the rest